Python 3.9.1 (tags/v3.9.1:1e5d33e, Dec  7 2020, 17:08:21) [MSC v.1927 64 bit (AMD64)] on win32
Type "help", "copyright", "credits" or "license()" for more information.
>>> 
=================== RESTART: D:/2 2/TS/Module 1/21_01_2021.py ==================
Employee Id: 1539 Employee Name KLU
>>> 
=================== RESTART: D:/2 2/TS/Module 1/21_01_2021.py ==================
Employee Id: 1539 Employee Name KLU
>>> 
=================== RESTART: D:/2 2/TS/Module 1/21_01_2021.py ==================
Non Parameteraised Constructor
>>> 
=================== RESTART: D:/2 2/TS/Module 1/21_01_2021.py ==================
Traceback (most recent call last):
  File "D:/2 2/TS/Module 1/21_01_2021.py", line 31, in <module>
    emp=Employee(19,"Covid")
TypeError: __init__() takes 1 positional argument but 3 were given
>>> 
=================== RESTART: D:/2 2/TS/Module 1/21_01_2021.py ==================
Parameteraised Constructor
Employee Id: 19 Employee Name: Covid
>>> 
=================== RESTART: D:/2 2/TS/Module 1/21_01_2021.py ==================
Traceback (most recent call last):
  File "D:/2 2/TS/Module 1/21_01_2021.py", line 41, in <module>
    obj1.view()
AttributeError: 'A' object has no attribute 'view'
>>> 
=================== RESTART: D:/2 2/TS/Module 1/21_01_2021.py ==================
Class A method
Class A method
>>> 
=================== RESTART: D:/2 2/TS/Module 1/21_01_2021.py ==================
Class A method
Class B method
Class A method
>>> 
=================== RESTART: D:/2 2/TS/Module 1/21_01_2021.py ==================
Class A method
Class B method
Class C method
Class A method
Class B method
Class A method
>>> 
=================== RESTART: D:/2 2/TS/Module 1/21_01_2021.py ==================
Class A method
Class B method
Class C method
Class B method
Class A method
>>> 
=================== RESTART: D:/2 2/TS/Module 1/21_01_2021.py ==================
Class A method
Class B method
Class C method
Class A method
Class A method
>>> 
=================== RESTART: D:/2 2/TS/Module 1/21_01_2021.py ==================
Class A method
Class B method
Class C method
Class D method
Class A method
Class B method
Class A method
Class B method
Class A method
>>> 
=================== RESTART: D:/2 2/TS/Module 1/21_01_2021.py ==================
Area 0
Area 25
Area 50
>>> 
=================== RESTART: D:/2 2/TS/Module 1/21_01_2021.py ==================
Class A Method
Traceback (most recent call last):
  File "D:/2 2/TS/Module 1/21_01_2021.py", line 156, in <module>
    s.area(5,10)
NameError: name 's' is not defined
>>> 
=================== RESTART: D:/2 2/TS/Module 1/21_01_2021.py ==================
Class A Method
Class B Method
>>> 
=================== RESTART: D:/2 2/TS/Module 1/21_01_2021.py ==================
Traceback (most recent call last):
  File "D:/2 2/TS/Module 1/21_01_2021.py", line 166, in <module>
    obj1=Student("KLU",40)
  File "D:/2 2/TS/Module 1/21_01_2021.py", line 163, in __init__
    Student.count = Student.count + 1
AttributeError: type object 'Student' has no attribute 'count'
>>> 
=================== RESTART: D:/2 2/TS/Module 1/21_01_2021.py ==================
Traceback (most recent call last):
  File "D:/2 2/TS/Module 1/21_01_2021.py", line 170, in <module>
    print("Total No of Obj Created = ",Student.count())
TypeError: 'int' object is not callable
>>> 
=================== RESTART: D:/2 2/TS/Module 1/21_01_2021.py ==================
Total No of Obj Created =  3
>>> 
=================== RESTART: D:/2 2/TS/Module 1/21_01_2021.py ==================
Traceback (most recent call last):
  File "D:/2 2/TS/Module 1/21_01_2021.py", line 182, in <module>
    cse = Department(1539,"KLU","VJA","CSE")
  File "D:/2 2/TS/Module 1/21_01_2021.py", line 180, in __init__
    super().__init__(id,name,location)
NameError: name 'name' is not defined
>>> 
=================== RESTART: D:/2 2/TS/Module 1/21_01_2021.py ==================
1539 KLU VJA CSE
>>> 
=================== RESTART: D:/2 2/TS/Module 1/21_01_2021.py ==================
Enter First Value5
Enter First Value5
>>> 
=================== RESTART: D:/2 2/TS/Module 1/21_01_2021.py ==================
Enter First Value5
Enter Second Value1
5.0
>>> 
=================== RESTART: D:/2 2/TS/Module 1/21_01_2021.py ==================
Enter First Value
=================== RESTART: D:/2 2/TS/Module 1/21_01_2021.py ==================
Enter First Value: 5
Enter Second Value: 0
divisible by zero exception
>>> 
=================== RESTART: D:/2 2/TS/Module 1/21_01_2021.py ==================
30
No Exception raised
>>> 
=================== RESTART: D:/2 2/TS/Module 1/21_01_2021.py ==================
sravya
No Exception raised
>>> 
=================== RESTART: D:/2 2/TS/Module 1/21_01_2021.py ==================
Variable is not defined
>>> 
=================== RESTART: D:/2 2/TS/Module 1/21_01_2021.py ==================
Variable is not defined
30
No Exception raised
>>> 
=================== RESTART: D:/2 2/TS/Module 1/21_01_2021.py ==================
Variable is not defined
Index Bound of Exception
>>> 
=================== RESTART: D:/2 2/TS/Module 1/21_01_2021.py ==================
Index Bound of Exception
>>> 
=================== RESTART: D:/2 2/TS/Module 1/21_01_2021.py ==================
Index Bound of Exception
>>> 
=================== RESTART: D:/2 2/TS/Module 1/21_01_2021.py ==================
Index Bound of Exception
>>> 
=================== RESTART: D:/2 2/TS/Module 1/21_01_2021.py ==================
Index Bound of Exception
50
End of Program
>>> 
=================== RESTART: D:/2 2/TS/Module 1/21_01_2021.py ==================
50
End of Program
>>> 
=================== RESTART: D:/2 2/TS/Module 1/21_01_2021.py ==================
50
End of Program
Index Bound of Exception
>>> 
=================== RESTART: D:/2 2/TS/Module 1/21_01_2021.py ==================
Index Bound of Exception
>>> 
=================== RESTART: D:/2 2/TS/Module 1/21_01_2021.py ==================
Enter Number2
NO Exception Occured
>>> 
=================== RESTART: D:/2 2/TS/Module 1/21_01_2021.py ==================
Enter Number-2
Traceback (most recent call last):
  File "D:/2 2/TS/Module 1/21_01_2021.py", line 254, in <module>
    raise Exception("Raised an Exception")
Exception: Raised an Exception
>>> 
=================== RESTART: D:/2 2/TS/Module 1/21_01_2021.py ==================
>>> import 20_01_2021
SyntaxError: invalid syntax
>>> 
=================== RESTART: D:/2 2/TS/Module 1/21_01_2021.py ==================
>>> import 21_01_2021
SyntaxError: invalid syntax
>>> import Module 1
SyntaxError: invalid syntax
>>> import os
>>> os.name
'nt'
>>> os.mkdir("d:\\2 2\\TS\\CC")
>>> os.getcwd()
'D:\\2 2\\TS\\Module 1'
>>> os.chdir("d:\\")
>>> os.getcwd()
'd:\\'
>>> os.rmdir("d:\\2 2\\TS\\CC")
>>> os.getcwd()
'd:\\'
>>> os.listdir()
['$RECYCLE.BIN', '.git', '2 1', '2 2', 'com', 'day4', 'eclipse', 'META-INF', 'oracle', 'org', 'Reference Globe Platform Login Credentials - Reg. KLMail.eml', 'Resume_CV.pdf', 'sample.txt', 'System Volume Information']
>>> import datetime
>>> obj = datetime.datetime.now()
>>> print(obj)
2021-01-21 11:54:09.773500
>>> d = datetime.date(2021,1,21)
>>> d
datetime.date(2021, 1, 21)
>>> print(d)
2021-01-21
>>> from datetime import date
>>> d = datetime(2021,1,21)
Traceback (most recent call last):
  File "<pyshell#19>", line 1, in <module>
    d = datetime(2021,1,21)
TypeError: 'module' object is not callable
>>> d = date(2021,1,21)
>>> print(d)
2021-01-21
>>> from datetime import date
>>> today = date.today()
>>> print("Current Date: "today)
SyntaxError: invalid syntax
>>> print("Current Date: ", today)
Current Date:  2021-01-21
>>> print(today.year, today.month, today.day)
2021 1 21
>>> from datetime import datetime
>>> now = datetime.now()
>>> print(now)
2021-01-21 11:59:37.682936
>>> current_time = now.strftime("%H:%M:%S") #strftime
>>> print(current_time)
11:59:37
>>> import time
>>> t = time.localtime()
>>> current_time = time.strftime("%H:%M:%S",t)
>>> print(current_time)
12:01:07
>>> 
================================ RESTART: Shell ================================
>>> #Random Module
>>> import random
>>> print(random.random())  #a random float no in between 0.0 and 1.0
0.7253337822892395
>>> random.randint(10,20)   #a random integer between 10 and 20
15
>>> l = [10,20,30,40,50]
>>> random.choice(l)
40
>>> random.shuffle(l)       #elements are shuffled randomly
>>> l
[30, 20, 50, 10, 40]
>>> random.randrange(1,10)  #10 is not included
5
>>> random.randrange(1000,100,-2)
778
>>> random.randrange(100,1000,3)
757
>>> random.choices(l,weights=[5,2,1,6,4],k=3)  #weights are optional
[10, 20, 30]
>>> 
================================ RESTART: Shell ================================
>>> import math
>>> math.factorial(2)
2
>>> math.gcd(5,10)
5
>>> math.pow(2,3)
8.0
>>> math.isfinite(2/3)
True
>>> math.isfinite(2/0)
Traceback (most recent call last):
  File "<pyshell#53>", line 1, in <module>
    math.isfinite(2/0)
ZeroDivisionError: division by zero
>>> math.isnan("klu")
Traceback (most recent call last):
  File "<pyshell#54>", line 1, in <module>
    math.isnan("klu")
TypeError: must be real number, not str
>>> math.isnan(2)
False
>>> 
================================ RESTART: Shell ================================
>>> __name__
'__main__'
>>> 
=================== RESTART: D:/2 2/TS/Module 1/21_01_2021.py ==================
in 21_01_2021.py
Area:  15.0
>>> 